dev/ctests/cases/C48_ATM_gfs_fcst_seg0.yaml

```
@@ -7,63 +7,250 @@
-        - ["{{ 'CTESTS_STAGED_TESTS_DIR' | getenv }}/COMROOT/{{ 'PSLOT' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf108.grib2", "{{ 'RUNTESTS' | getenv }}/COMROOT/{{ 'TEST_NAME' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf108.grib2"]
-        - ["{{ 'CTESTS_STAGED_TESTS_DIR' | getenv }}/COMROOT/{{ 'PSLOT' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf111.grib2", "{{ 'RUNTESTS' | getenv }}/COMROOT/{{ 'TEST_NAME' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf111.grib2"]
-        - ["{{ 'CTESTS_STAGED_TESTS_DIR' | getenv }}/COMROOT/{{ 'PSLOT' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf114.grib2", "{{ 'RUNTESTS' | getenv }}/COMROOT/{{ 'TEST_NAME' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf114.grib2"]
-        - ["{{ 'CTESTS_STAGED_TESTS_DIR' | getenv }}/COMROOT/{{ 'PSLOT' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf117.grib2", "{{ 'RUNTESTS' | getenv }}/COMROOT/{{ 'TEST_NAME' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf117.grib2"]
-        - ["{{ 'CTESTS_STAGED_TESTS_DIR' | getenv }}/COMROOT/{{ 'PSLOT' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf120.grib2", "{{ 'RUNTESTS' | getenv }}/COMROOT/{{ 'TEST_NAME' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t12z.sfluxgrbf120.grib2"]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.diag_table, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.diag_table]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.input.nml, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.input.nml]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.model_configure, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.model_configure]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.ufs.configure, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.ufs.configure]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf000.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf000.txt]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf003.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf003.txt]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf006.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf006.txt]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf009.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf009.txt]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf0{{ cyc }}.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf0{{ cyc }}.txt]
```

Same as my comments in the other yaml...combine lines with loops over forecast hour and/or replace these instances of `{{ cyc }}` with `12`. I am not flagging each `{{ cyc }}` line in here but you get the point. :)
```suggestion
        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf012.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf012.txt]
```

---

Yes I will look into how to channel {{ cyc }} and discuss the loop approach with Rahul and see if he would like to reconsider adopting it.

---

Sounds good, thanks @TerrenceMcGuinness-NOAA !